include/hw_unique_key.h

```diff
@@ -30,10 +30,10 @@ extern "C" {
 #if DT_HAS_COMPAT_STATUS_OKAY(nordic_nrf_kmu)
 #define HUK_HAS_KMU
 #endif
-#if defined(CONFIG_HAS_HW_NRF_CC310)
+#if defined(CONFIG_CRYPTOCELL_CC310_USABLE)
```

Comment:
Because the HAS_HW_NRF_CC310 is not available anymore for TF-M builds. The new PSA driver from Zephyr has a compatible text "zephyr,psa-crypto-rng".

Reply:
```
config HAS_HW_NRF_CC310
	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF_CC310)) || $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF_CC310_SW))
```

So this should perhaps be updated?

Are we not using the cryptocell_sw node anymore?

Reply:
I will investigate if this is the possible, maybe we can eliminate the "USABLE" kconfig.